dojo/tools/wazuh/parser.py

```diff
@@ -0,0 +1,89 @@
+import json
+from dojo.models import Finding
+
+
+class WazuhParser(object):
+    """
+    Use Wazuh Vulnerability API to retrieve the findings
+    The vulnerabilities with condition "Package unfixed" are skipped because there is no fix out yet.
+    https://github.com/wazuh/wazuh/issues/14560
+    """
+
+    def get_scan_types(self):
+        return ["Wazuh"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "Wazuh"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Wazuh"
+
+    def get_findings(self, filename, test):
+        tree = filename.read()
+        try:
+            data = json.loads(str(tree, "utf-8"))
+        except:
+            data = json.loads(tree)
+
+        # Detect duplications
+        dupes = dict()
+
+        try:
+            vulnerability = data[next(iter(data.keys()))]["affected_items"]
+        except (KeyError, StopIteration):
+            return list()
+
+        if vulnerability is None:
+            return list()
+
+        for item in vulnerability:
+            if item['condition'] != "Package unfixed":
+                id = item.get('cve')
+                package_name = item.get('name')
+                package_version = item.get('version')
+                description = item.get('condition')
+                severity = item.get('severity')
+                links = item.get('external_references')
+                title = item.get('title') + " (version: " + package_version + ")"
+                severity = transpose_severity(severity)
+                if links:
+                    references = ''
+                    for link in links:
+                        references += f'{link}\n'
+                else:
+                    references = None
+
+                if id and id.startswith('CVE'):
+                    vulnerability_id = id
+                else:
+                    vulnerability_id = None
+
+                dupe_key = title
+
+                if dupe_key in dupes:
+                    find = dupes[dupe_key]
+                else:
+                    dupes[dupe_key] = True
+
+                    find = Finding(
+                        title=title,
+                        test=test,
+                        description=description,
+                        severity=severity,
+                        mitigation="mitigation",
+                        references=references,
+                        static_finding=True,
+                        component_name=package_name,
+                        component_version=package_version,
+                    )
+                    if vulnerability_id:
+                        find.unsaved_vulnerability_ids = [vulnerability_id]
+                    dupes[dupe_key] = find
+        return list(dupes.values())
+
+
+def transpose_severity(severity):
+    if severity in Finding.SEVERITIES:
+        return severity
+    else:
+        return 'Info'
```

Comment:
This silently mask new severity data that will be different. It's not good.

Reply:
Ok, how can I do this better? I have looked at: 
https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/harbor_vulnerability/parser.py 
This is how it is done there as well.

Reply:
A simple form like this is better:
```Python
severity = item.get('severity', 'Info')
```

Reply:
Ah right. Thank you for the hint.